modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -84,9 +86,9 @@
 
 		&-title {
 			display: flex;
-			align-items: var( --n-tabs-title-align-items );
+			align-items: var( --n-tabs-title-align-items-toggle, var( --n-tabs-title-align-items ) );
```

Comment:
Why do we need the fallback if `--n-tabs-title-align-items-toggle` always contains a value?
I see that `--n-tabs-title-align-items-toggle` is set in row 42.

Reply:
@davseve Yes, we assign `initial` to `--n-tabs-title-align-items-toggle` it use the fallback value instead.